### PR TITLE
Add comment feature

### PR DESCRIPTION
**Pull request by winebarrel**

### CODE REVIEW

File: db/migrate/20160907123728_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table "comments", force: :cascade do |t|
+      t.integer  "incident_id", limit: 4,   null: false
+      t.integer  "user_id",     limit: 4,   null: false
+      t.string   "comment",     limit: 255, null: false
```
Comment (ryotarai) [inline db/migrate/20160907123728_create_comments.rb]: I think `comment` can be longer then 255. How about changing this to text column?

Comment (winebarrel) [inline db/migrate/20160907123728_create_comments.rb]: Thanks! Fix it c821ae4

**Comment by winebarrel** [comment]
@ryotarai ping

**Comment by ryotarai** [comment]
Thank you for your contribution.

**Comment by winebarrel** [comment]
Thanks!

**Pull request merged**